notanorm/model.py

```diff
@@ -48,11 +48,12 @@ def __eq__(self, other):
 class DbIndex(NamedTuple):
     """Index definition."""
     fields: Tuple[str, ...]         # list of fields in the index
+    field_lengths: Tuple[Optional[int], ...] = ()
```

Comment:
probably should just go ahead and make a DbIndexField tuple.   would prevent some mistakes i think

Reply:
Done, added a new object type for `DbIndexField`s